packages/design-system/src/Popover/Popover.stories.tsx

```diff
@@ -64,6 +64,7 @@ const PopoverContentTemplate: ComponentStory<typeof PopoverHeader> = (args) => {
   );
 };
 
+//  TODO: The popover guidelines say that popovers should have at least one focusable element in it, but this story doesn't because PopoverHeader expects children to be a string. Change this.
```

Comment:
When are we planning for this? Can you create a task for this.

Reply:
We haven't planned for this yet. Sure, I'll create a task.

Reply:
We haven't planned for this yet. Sure, I'll create a task. https://github.com/appsmithorg/appsmith/issues/26844

Reply:
https://github.com/appsmithorg/appsmith/issues/26844